playbook.yml

```diff
@@ -449,7 +429,6 @@
       omero.glacier2.IceSSL.CertFile: server.p12
       # This password doesn't need to be secret
       omero.glacier2.IceSSL.Password: secret
-      omero.fs.repo.path: "%user%_%userId%/%thread%//%year%-%month%/%day%/%time%"
```

Comment:
This isn't directly tied to dropbox, but it's also ok to remove. It only affects where _new_ imports are placed.

Reply:
ok, but there might be new import from late data curation (files that remained to import at the end of the workshop), or images converted to ome.tiff that would be imported again

Reply:
but if you are looking to have the layout, it depends on what your omero.fs.repo.path at **TiM** was not here on the IDR. (This repo.path setting is optimized to prevent concurrency issues in creating the date-stamped directories)

Reply:
my config was the same (I also used that playbook to make the TIM server)